README.md

```diff
@@ -1,10 +1,28 @@
 A Time Tracker 
 ==============
-Open Source Time Tracker for Android
+*Open Source Time Tracker for Android*
 
 
 - Based on TimeTracker+ by Sean E. Russell.
 - Application icon from Tango Project (tango.freedesktop.org)
 - Russian translation by Denis Gladkiy
 - French translation by Wilfried Kopp
 - Spanish and Italian translation by Carlos Manzo
+
+### Source code on [github](http://github.com/netmackan/ATimeTracker)
```

Comment:
Me being a bit pedantic...:
- "github" -> "GitHub"
- Use https:// for the link